serving/src/main/java/ai/djl/serving/http/ClusterRequestHandler.java

```diff
@@ -101,9 +129,30 @@ protected void handleRequest(
         }
     }
 
+    private void restartSshServer() {
```

Comment:
Do we really need restart sshserver?

Reply:
It is recommended to restart server after modifying keys, but it seems that's unnecessary unless we make changes to the config, removed it.